dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -167,6 +169,8 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
             AppNotificationTelemetry::LogRegister(hr, m_appId);
         }) };
 
+        // Need to clear the RoActivateInstance caching for LRP proxyStub to be found.
+        PushNotificationHelpers::ClearRoActivateInstanceCache();
```

Comment:
Instead of doing this everytime you call CoCreateIntance on LongRunningPlatform, you could do the following.

1) Create a utility method called GetLongRunningPlatform.
2) The first thing this does would be to clear the cache for RoActivate
3) It would then CoCreate a new platform and return it.

By doing this, you have one single method that all components can reuse and a single place for the mitigation.

Reply:
This is under discussion in other thread.